packages/core/src/Keymap.ts

```diff
@@ -17,13 +17,64 @@ interface ShortcutItem {
     priority: number;
 }
 
+/**
+ * Keymap allow to add and remove shortucts and provide a function to match if a
+ * particular pattern is found within registred shortucts.
+ *
+ * The expression to describe a shortuct is zero or more `modifiers` and one
+ * `hotkey` joined with the symbol `+`.
+ *
+ * ## Modifiers
+ * - SHIFT
+ * - ALT
+ * - CTRL
+ * - META
+ * - CMD (alias of META)
+ *
+ * Example:
+ * ```typescript
+ * // using a modifier and one key
+ * keymap.addShortcut('CTRL+A', 'a-command')
+ * // using multiples modifiers and one key
+ * keymap.addShortcut('CTRL+ALT+A', 'a-command')
```

Comment:
I would write "commandIdentifier" rather than "a-command" because I had to go back to the signature of the `addShortcut` function to understand what that was supposed to be.